mlir/docs/TargetLLVMIR.md

```diff
@@ -290,6 +290,21 @@ memref<2 x vector<4x8 x f32>
 Tensor types cannot be converted to the LLVM dialect. Operations on tensors must
 be [bufferized](Bufferization.md) before being converted.
 
+### Conversion of LLVM Container Types with Non-Compatible Element Types
+
+Progressive lowering may result in there existing LLVM container types, such
```

Comment:
```suggestion
Progressive lowering may result in existing LLVM container types, such
```
nit: I believe the there should not be there?

Reply:
You accidentally deleted the wrong word, I think.